File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int getHeight()
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (1 > height || height > 8);
    return height;
}

int main(void)
{
    int j = 0;
    //getting height from user 1 to 8 exclusive
    int height = getHeight();
    for (int i = 0 ; i < height ; i++)
    {
        //printing the spaces
        for (j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        //printing the tag
        for (j = 0; j <= i; j++)
        {
            printf("#");
        }
        //space in between
        printf("  ");
        //second half
        for (j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# Prompt the user for a valid height value
height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

i = int(1)

while i <= height:
    print((" " * (height-i)) + ("#" * i) + "  " + ("#"*i))
    i += 1